eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -21,6 +21,9 @@ stages:
                 NonWindowsPath: ./build/sdk/samples/iot/paho_iot_hub_sas_telemetry_sample
                 WindowsPath: .\build\sdk\samples\iot\Debug\paho_iot_hub_sas_telemetry_sample.exe
                 RunCondition: contains(variables['build.args'], '-DTRANSPORT_PAHO=ON')
+              - Name: Storage Blobs
+                NonWindowsPath: ./build/sdk/samples/storage/blobs/blobs_client_sample
+                WindowsPath: build\sdk\samples\storage\blobs\debug\blobs_client_sample.exe
```

Comment:
Can we try triggering and running the storage blobs live sample tests in this PR to verify this change works as expected?

cc @danieljurek

Reply:
I ran the sample locally, looks good.